assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -236,10 +245,12 @@ function MediaPane(props) {
   const resources = media.filter(filterResource);
 
   const onSearch = (value) => {
-    setSearchTerm({ searchTerm: value });
-    trackEvent('search_media', 'editor', '', '', {
-      search_term: value,
-    });
+    if (value.trim() !== searchTerm) {
+      setSearchTerm({ searchTerm: value.trim() });
+      trackEvent('search_media', 'editor', '', '', {
+        search_term: value.trim(),
```

Comment:
Looks like there are 4x `trim()` calls here.

Reply:
Merged to a variable.